src/gather/api.py

```diff
@@ -176,29 +198,31 @@ def raise_if_needed(self):
         if self._please_raise is not None:
             raise self._please_raise
 
+
 def run(argv, commands, version, output):
-    """Run the correct subcommand.
-
-    :param argv: Arguments to be processed
-    :type argv: List of strings
-    :param commands: Commands (usually collected by a :code:`Collector`)
-    :type commands: Mapping of strings to functions that accept arguments
-    :param str version: Version string to display
-    :param file output: Where to write output to
+    """
+    Run the correct subcommand.
+
+    Args:
+        argv (list of str): Arguments to be processed
+        commands (mapping of str to callables): Commands (usually collected by a :code:`Collector`)
+        version (str): Version to display if :code:`--version` is asked
+        output (file): Where to write output to
     """
     if len(argv) < 1:
         argv = argv + ['help']
     if argv[0] in ('version', '--version'):
-        print("Version {}".format(version), file=output)
+        output.write("Version {}\n".format(version))
```

Comment:
If you no longer use the `print` function in this module, you can remove [the future flag at the top of the file](https://github.com/moshez/gather/pull/17/files#diff-cf6c32c495bfb435cbe20041c1ae806fR24).